data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
```

Comment:
My take on this is that one should make as few assumptions about what order platforms implement ES6 features as possible. So, I think this should use the `__proto__` setter instead of `Object.setPrototypeOf`, on the basis that it *technically* predates ES6 itself, and has the widest support base.